sdk/main/include/impl/Node.h

```diff
@@ -50,79 +53,169 @@ class Node
+   */
+  grpc::Status submitQuery(proto::Query::QueryCase funcEnum,
+                           const proto::Query& query,
+                           const std::chrono::system_clock::time_point& deadline,
+                           proto::Response* response);
+
+  /**
+   * Submit a Transaction to this Node.
+   *
+   * @param funcEnum    The enumeration specifying which gRPC function to call for this specific Transaction.
+   * @param transaction The Transaction protobuf object to send.
+   * @param deadline    The deadline for submitting this Transaction.
+   * @param response    Pointer to the TransactionResponse protobuf object to fill with the gRPC server's response.
+   * @return The gRPC status response of the function call from the gRPC server.
    */
-  [[nodiscard]] inline std::shared_ptr<AccountId> getAccountId() const { return mAddress->getAccountId(); }
+  grpc::Status submitTransaction(proto::TransactionBody::DataCase funcEnum,
+                                 const proto::Transaction& transaction,
+                                 const std::chrono::system_clock::time_point& deadline,
+                                 proto::TransactionResponse* response);
 
   /**
-   * Sets the TLS behavior of the node
+   * Set the TLS behavior this Node should utilize.
    *
-   * @param desiredBehavior the desired behavior
+   * @param desiredBehavior The desired behavior.
    */
   void setTLSBehavior(TLSBehavior desiredBehavior);
 
   /**
-   * Get a gRPC transaction method for an associated protobuf Transaction data case from this Node's channel.
+   * Set the minimum amount of time this Node should wait before being willing to resubmit a previously failed request.
    *
-   * @param transactionBodyDataCase The case describing the function to get.
-   * @return The function described by the case, bound to this channel's proper stub.
+   * @param backoff The minimum backoff time.
    */
-  std::function<grpc::Status(grpc::ClientContext*, const proto::Transaction&, proto::TransactionResponse*)>
-  getGrpcTransactionMethod(int transactionBodyDataCase);
+  void setMinBackoff(const std::chrono::duration<double>& backoff);
 
   /**
-   * Get a gRPC query method for an associated protobuf Query data case from this Node's channel.
+   * Set the maximum amount of time this Node should wait before being willing to resubmit a previously failed request.
    *
-   * @param queryBodyDataCase The case describing the function to get.
-   * @return The function described by the case, bound to this channel's proper stub.
+   * @param backoff The maximum backoff time.
    */
-  std::function<grpc::Status(grpc::ClientContext*, const proto::Query&, proto::Response*)> getGrpcQueryMethod(
-    int queryBodyDataCase);
+  void setMaxBackoff(const std::chrono::duration<double>& backoff);
 
-private:
   /**
-   * The address of this node.
+   * Determine if this Node is "healthy", meaning this Node has not received any bad gRPC statuses, or if it has then it
```

Comment:
Since it can mean a variety of things (from my understanding at least), I changed up the wording a bit to just say successful. Let me know if you think it's descriptive enough.